Makefile

```diff
@@ -35,6 +35,9 @@ imgs_server_dest := $(imgs_source:%=build/server/%)
 
 raven_source := $(shell node -e 'console.log(require.resolve("raven-js/dist/raven.js"))')
 
+l10n_source := $(wildcard locales/*/*.properties)
+l10n_dest := $(l10n_source:%/*.properties=addon/webextension/_locales/%/messages.json)
```

Comment:
not sure about this, but `make` doesn't seem to complain, so I guess it's alright?